common/Dockerfile.tests

```diff
@@ -0,0 +1,19 @@
+FROM python:3.6.1-slim
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+COPY common/requirements.txt .
+
+RUN pip install -r requirements.txt
+
+COPY data_models/dist/data-refinery-models-* data_models/
+
+# Get the latest version from the dist directory.
+RUN pip install data_models/$(ls data_models -1 | tail -1)
```

Comment:
Hmmm are you sure this is always going to return the latest version? I think it might not, since (for example) `ls` lists version `1.10` first, before `1.9`, so `1.9` would be your last version listed on the dist directory.